### ISSUE TITLE
Tachiyomi keeps stopping on launch

### ISSUE THREAD

**Issue by CrimsonRuby11** [tachiyomiorg/tachiyomi#6846]
### Steps to reproduce

1.) install tachiyomi app
2.) use for a while
3.) leave it unopened for a few hours and then try to open it

### Expected behavior

App should open normally

### Actual behavior

Tachiyomi keeps stopping even after clearing cache, data, and also restarting device.

### Crash logs

I can't access the logs section of the app because when i reinstall it the crash logs starts fresh... however i did perform a logcat so i'll put that here...
```
04-02 10:15:10.241 17038 17048 I art     : Enter while loop.
04-02 10:15:10.244 17038 17038 E ResourceType: Style contains key with bad entry: 0x0101056c
04-02 10:15:10.249 17038 17038 E ResourceType: Style contains key with bad entry: 0x0101056d
04-02 10:15:10.249 17038 17038 E ResourceType: Style contains key with bad entry: 0x0101058c
04-02 10:15:10.249 17038 17038 E ResourceType: Style contains key with bad entry: 0x01010604
04-02 10:15:10.249 17038 17038 E ResourceType: Style contains key with bad entry: 0x01010605
04-02 10:15:10.256 17038 17038 E ResourceType: Style contains key with bad entry: 0x0101056d
04-02 10:15:10.256 17038 17038 E ResourceType: Style contains key with bad entry: 0x0101058c
04-02 10:15:10.256 17038 17038 E ResourceType: Style contains key with bad entry: 0x01010604
04-02 10:15:10.256 17038 17038 E ResourceType: Style contains key with bad entry: 0x01010605
04-02 10:15:10.267 17038 17038 F libc    : Fatal signal 11 (SIGSEGV), code 1, fault addr 0x19 in tid 17038 (anade.tachiyomi)
04-02 10:15:10.267   488   488 D AEE/AED : $===AEE===AEE===AEE===$
04-02 10:15:10.267   488   488 D AEE/AED : p 2 poll events 1 revents 1
04-02 10:15:10.267   488   488 D AEE/AED : aed_main_fork_worker: generator 0xa8594dc8, worker 0xbecc4958, recv_fd 0
04-02 10:15:10.269 17064 17064 I AEE/AED : handle_request(0)
04-02 10:15:10.269 17064 17064 I AEE/AED : check process 17038 name:anade.tachiyomi
04-02 10:15:10.269   488   488 D AEE/AED : aee handle:-1, cpu cores:4, online:4
04-02 10:15:10.269 17064 17064 I AEE/AED : tid 17038 abort msg address:0x00000000, si_code:1 (request from 17038:10355)
04-02 10:15:10.269 17064 17064 W AEE/AED : debuggerd: handling request: pid=17038 uid=10355 gid=10355 tid=17038
04-02 10:15:10.271 17064 17064 I AEE/AED : [preset_info] pid: 17038, tid: 17038, name: anade.tachiyomi  >>> eu.kanade.tachiyomi <<<
04-02 10:15:10.342 17064 17064 I AEE/AED : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
04-02 10:15:10.342 17064 17064 I AEE/AED : Build fingerprint: 'motorola/panell_21_d/panell_d:7.0/NRD90M.03.045/045:user/release-keys'
04-02 10:15:10.342 17064 17064 I AEE/AED : Revision: '0'
04-02 10:15:10.342 17064 17064 I AEE/AED : ABI: 'arm'
04-02 10:15:10.342 17064 17064 I AEE/AED : pid: 17038, tid: 17038, name: anade.tachiyomi  >>> eu.kanade.tachiyomi <<<
04-02 10:15:10.343 17064 17064 I AEE/AED : signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x19
04-02 10:15:10.343 17064 17064 I AEE/AED :     r0 00002c42  r1 00000001  r2 00000000  r3 12c959b0
04-02 10:15:10.343 17064 17064 I AEE/AED :     r4 0026f7ad  r5 00000001  r6 00000000  r7 137bd680
04-02 10:15:10.343 17064 17064 I AEE/AED :     r8 00000002  r9 ac985400  sl 137c9cd0  fp 12c959b0
04-02 10:15:10.343 17064 17064 I AEE/AED :     ip 12c95000  sp bef33650  lr 9345ddf9  pc ac53fa50  cpsr 000d0030
04-02 10:15:10.345 17064 17064 I AEE/AED :
04-02 10:15:10.345 17064 17064 I AEE/AED : backtrace:
04-02 10:15:10.346 17064 17064 I AEE/AED :     #00 pc 000ada50  /system/lib/libart.so
04-02 10:15:10.346 17064 17064 I AEE/AED :     #01 pc 0130fdf7  /data/app/eu.kanade.tachiyomi-1/oat/arm/base.odex (offset 0x1095000)
04-02 10:15:10.467   903  1416 W PackageManager: checkUidPermission(): android.permission.INTERACT_ACROSS_USERS_FULL of 10352 is denied.
04-02 10:15:10.610  1640  1655 D GasService: FG app changed: from com.motorola.launcher3 to eu.kanade.tachiyomi
04-02 10:15:10.610  1640  1655 D GasService: irregular AudioTrack usage
```

### Tachiyomi version

0.13.1

### Android version

Android 7.0

### Device

Motorola Moto C Plus

### Other details

This started happening not a long time ago, Clearing cache, data, restarting device, or uninstalling or changing extensions does not work. I cleared out all my downloaded manga too, yet this still occurs. Only way to get back to my manga is to reinstall the app and restore backups if i've made any... I've tried reinstalling the chrome webview, didnt work. I hope this will be fixed soon, i've been enjoying the app for a long time... Thanks in advance!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by GodTaisaku** [comment]
I've also been having this problem since i updated it to the latest version

**Comment by Sakko-Nuuru** [comment]
This is happening to me too.

**Comment by Sakko-Nuuru** [comment]
@GodTaisaku do you also have Android 7.0?

**Comment by CrimsonRuby11** [comment]
Hi @Sakko-Nuuru
Just a small info, might be useful. The daily library updates still runs on the background even thought the app keeps crashing / stopping when opened. For now I'm testing version 0.12.3 if it works well for atleast a couple of days... 

Another thing my phone's battery is super weak so it ends up running out without me noticing most of the time (because im almost always on the pc...) so maybe something's getting corrupt due to unexpected reboots?

**Comment by Sakko-Nuuru** [comment]
> Hi @Sakko-Nuuru Just a small info, might be useful. The daily library updates still runs on the background even though the app keeps crashing / stopping when opened.

Yeah i still got notifications for library updates so I used the extension's website to read manga. I'm using 0.11.1 starting from today and i will try to see if it crashes there too. Since we both have android 7 this might be an android 7 specific issue. I really hope this gets fixed soon.

**Comment by Sakko-Nuuru** [comment]
The devs should fix this asap.

**Comment by Battyjuice** [comment]
I updated to version 13.2 just this morning and Tachiyomi hasn't worked since.

**Comment by CarlosEsco** [comment]
Do any of you have obsolete extensions installed?

**Comment by Battyjuice** [comment]
None at all.

**Comment by arkon** [comment]
Judging from the reports thus far this is specific to Android 7. I (or somebody) will need to dig more into it at some point, but it'll take a while for me to get around to it since I can't readily get an Android 7 emulator working on my current computer.

**Comment by arkon** [comment]
This doesn't seem to happen in an emulator, at least for now. Does this only happen after some time for everyone that's been affected?

**Comment by arkon** [comment]
Also did 0.13.1 work fine for anyone having these issues?

**Comment by CrimsonRuby11** [comment]
> This doesn't seem to happen in an emulator, at least for now. Does this only happen after some time for everyone that's been affected?

Yes it takes sometime, usually after a day or by the end of the day.

> Also did 0.13.1 work fine for anyone having these issues?

No, I was using version 0.13.1 when i first opened the issue, updating to 0.13.2 didnt work either

**Comment by GodTaisaku** [comment]
> Also did 0.13.1 work fine for anyone having these issues?

0.13.1 works for me

**Comment by GodTaisaku** [comment]
> Also did 0.13.1 work fine for anyone having these issues?

0.13.1 works for me

**Comment by arkon** [comment]
@GodTaisaku @Sakko-Nuuru @Battyjuice Are yours also only crashing after around a day, or at launch?

Can you all also post crash logs if possible?

Potential ways of getting them:

- More → Settings → Advanced → Dump crash logs
- https://help.getpocket.com/article/1073-retrieving-a-pocket-crash-log-on-android
- https://play.google.com/store/apps/details?id=com.dp.logcatapp&hl=en_CA

**Comment by arkon** [comment]
Unfortunately I can't create an API 23 (Android 6) emulator on an ARM Mac. Does it seem to happen on Android 7 for you as well?

**Comment by FourTOne5** [comment]
Android 6 user no crash on Stable 0.13.1+ or Preview r4224+ while using MangaSee. Your MangaSee extension probably became corrupted in some way did you try reinstalling?

**Comment by arkon** [comment]
@helloworld-1313 Can you try with [r4256](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r4256)?

**Comment by helloworld-1313** [comment]
> Unfortunately I can't create an API 23 (Android 6) emulator on an ARM Mac. Does it seem to happen on Android 7 for you as well?

i tried few android 7 emulators (nox, ld player) no issue. 



> @helloworld-1313 Can you try with [r4256](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r4256)?

tried r4256 on android 6 emulator (mumu) same issue.



> Android 6 user here no crash on Stable 0.13.1+ or Preview r4224+ while using MangaSee. Your MangaSee extension probably became corrupted in some way did you try reinstalling?

yes, already try reinstalling mangasee extension. same issue but it work on stable 0.13.1 and preview r4215 or earlier
mangadex extension dont have this issue


**Steps to reproduce**

scenario 1
- install tachiyomi > install mangasee extension > open tachiyomi app > click on "browse" tab > click "mangasee" to view catalog
it freeze / not responding (refer picture below)

![1](https://user-images.githubusercontent.com/103625770/163537539-4af3ff26-6dd8-44c2-a958-f32b57f4ea82.PNG)


scenario 2
- install tachiyomi > install mangasee extension > open tachiyomi app > restore from backup > click on "library" tab
it freeze / not responding (refer picture above) if library contain manga from mangasee source


**crash log from logcat on r4256 (scenario 1)**
[logcat_04-15-2022_02-29-41.txt](https://github.com/tachiyomiorg/tachiyomi/files/8494399/logcat_04-15-2022_02-29-41.txt)

**Comment by FourTOne5** [comment]
It works for me now and it always worked. Your problem is happening on emulators does the same happen on physical devices? It might be that emulator's issue since it worked on others. Also your issue seems to different from this issue since your one happens when using extension so make a new issue.

**Comment by CrimsonRuby11** [comment]
yeah in my case I can't open the app itself, I myself have tried migrating my manga between several extensions (mangadex to mangapark to mangasee)

I have a small suspicion that the problem might be because of corrupted backups? because every time I reinstall the app, I'd restore from the backup file... If the others also had problems after restoring from backups then maybe the problem lies on corrupted backup files... I'm testing version 0.13.1 without restoring backups on my android 7 device now... will update if it works.

**Comment by GodTaisaku** [comment]
I just downloaded 0.13.2 and it works perfectly for me now. Seems like the devs fixed it

**Comment by Sakko-Nuuru** [comment]
> I just downloaded 0.13.2 and it works perfectly for me now. Seems like the devs fixed it

I will try to update to 0.13.2 and see if it works. I'll edit this comment with an update 1 or 2 days later.

**Comment by arkon** [comment]
Can people having this issue try with [r4263](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r4263)?

**Comment by helloworld-1313** [comment]
> Can people having this issue try with [r4263](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r4263)?

nice. this update works for me

**Comment by arkon** [comment]
This _should_ be fixed in [v0.13.3](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.13.3).

**Comment by arkon** [comment]
Closing this for now. If there's still issues, open another one with newer sets of details/logs so it's easier to track.
